### ISSUE TITLE
Syllables are incorrectly counted

### ISSUE THREAD

**Issue by Anaphase** [cgiffard/TextStatistics.js#4]
Given the sentence:

> This sentence has eight syllables.

syllableCount() will return 9. As verified on https://readability-score.com, DaveChild/Text-Statistics returns the correct syllable count of 8.

**Comment by giorgio79** [comment]
Maybe you can take a cue from https://github.com/brbcoding/Readability

**Comment by JohnMcLear** [comment]
```
var stats = textstatistics("monkey goes yay");
stats.wordCount();
4
```
``-_-``

**Comment by cgiffard** [comment]
Fixed in #13. Sorry for neglecting this issue for so long!
